File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int pCnt = 0;
        int yCnt = 0;
        
        for(int i = 0; i<s.length(); i++){
            if(s.charAt(i) == 'P' || s.charAt(i) == 'p'){
                pCnt += 1;
            }else if(s.charAt(i) == 'Y' || s.charAt(i) == 'y'){
                yCnt += 1;
            }
        }
        
        if(pCnt == yCnt){
            return true;
        }else{
            return false;
        }
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    pCnt = 0
    yCnt = 0
    for ch in s:
        if ch == 'P' or ch == 'p':
            pCnt += 1
        elif ch == 'Y' or ch =='y':
            yCnt += 1
    if pCnt == yCnt:
        return True
    else:
        return False